medallion/backends/mongodb_backend.py

```diff
@@ -49,6 +64,12 @@ def __init__(self, **kwargs):
             super(MongoBackend, self).__init__(**kwargs)
             self.pages = {}
             self.client = MongoClient(kwargs.get("uri"))
+
+            if kwargs.get("filename"):
+                log.info("Initializing Mongo DB backend using " + kwargs.get("filename"))
+                self._clear_db()
```

Comment:
This means every time the server is started up, the db will be cleared, right? Users may want data to persist if they stop the server and start it up again later. It should definitely be cleared in our tests, though.

Reply:
So my assumption is that you only need to clear the db if you are initializing it.  If you aren't initializing it, then you wouldn't have the "filename" property in the config file.  Its a little clunky - do you have a better suggestion?

Reply:
I'm assuming people wouldn't change their config file after initializing it.
My suggestion is to remove this line and instead call `_clear_db()` in `TaxiiTest.setUp`.
And maybe make it a public method?

Reply:
Perhaps, an additional config property (e.g. initialize_db: True) or a command-line argument